src/pq_flash_index.cpp

```diff
@@ -507,7 +507,7 @@ namespace diskann {
       _u32 &num_lbls_in_cur_pt = _pts_to_labels[counter];
       num_lbls_in_cur_pt = 0;
       counter++;
-      getline(iss, token, '\t');  // first token contains metadata, not used
+      //getline(iss, token, '\t');  // first token contains metadata, not used
```

Comment:
We can remove this commented line? It appears to be a duplicate of the following, un-commented line of code